### DISCUSSION TITLE
添加模组迁移功能

### DISCUSSION THREAD

**Discussion by EagleJinjkads** [Meloong-Git/PCL#4257] [提问] [answered]
可以选择任意版本，使用模组迁移动功能，选择一个目标版本，然后自动检查所有在PCL内下载的模组与目标版本的兼容性，将兼容的模组自动全部下载至目标版本内，然后再弹窗提示不兼容的那些以及自动安装了那些模组

**Comment by Pigeon0v0** [comment, marked as answer]
#1053 类似
